<?xml version="1.0" encoding="UTF-8"?>
<testng-results skipped="0" failed="2" total="4" passed="2">
  <reporter-output>
  </reporter-output>
  <suite name="Default suite" duration-ms="16245" started-at="2015-12-20T20:20:43Z" finished-at="2015-12-20T20:20:59Z">
    <groups>
    </groups>
    <test name="Default test" duration-ms="16245" started-at="2015-12-20T20:20:43Z" finished-at="2015-12-20T20:20:59Z">
      <class name="com.example.tests.CreateContact">
        <test-method status="PASS" signature="setUp()[pri:0, instance:com.example.tests.CreateContact@2ac1fdc4]" name="setUp" is-config="true" duration-ms="10032" started-at="2015-12-20T22:20:43Z" finished-at="2015-12-20T22:20:53Z">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- setUp -->
        <test-method status="PASS" signature="testCreateEmptyContact()[pri:0, instance:com.example.tests.CreateContact@2ac1fdc4]" name="testCreateEmptyContact" duration-ms="3148" started-at="2015-12-20T22:20:53Z" finished-at="2015-12-20T22:20:56Z">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- testCreateEmptyContact -->
        <test-method status="PASS" signature="testCreateNotEmptyContact()[pri:0, instance:com.example.tests.CreateContact@2ac1fdc4]" name="testCreateNotEmptyContact" duration-ms="2527" started-at="2015-12-20T22:20:56Z" finished-at="2015-12-20T22:20:58Z">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- testCreateNotEmptyContact -->
        <test-method status="PASS" signature="tearDown()[pri:0, instance:com.example.tests.CreateContact@2ac1fdc4]" name="tearDown" is-config="true" duration-ms="499" started-at="2015-12-20T22:20:58Z" finished-at="2015-12-20T22:20:59Z">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- tearDown -->
      </class> <!-- com.example.tests.CreateContact -->
      <class name="com.example.tests.CreateGroup">
        <test-method status="FAIL" signature="testEmptyTestCreation()[pri:0, instance:com.example.tests.CreateGroup@71f2a7d5]" name="testEmptyTestCreation" duration-ms="4" started-at="2015-12-20T22:20:58Z" finished-at="2015-12-20T22:20:58Z">
          <exception class="java.lang.NullPointerException">
            <full-stacktrace>
              <![CDATA[java.lang.NullPointerException
	at com.example.tests.CreateGroup.testEmptyTestCreation(CreateGroup.java:22)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
	at java.lang.reflect.Method.invoke(Method.java:497)
	at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:85)
	at org.testng.internal.Invoker.invokeMethod(Invoker.java:639)
	at org.testng.internal.Invoker.invokeTestMethod(Invoker.java:821)
	at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:1131)
	at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:124)
	at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:108)
	at org.testng.TestRunner.privateRun(TestRunner.java:773)
	at org.testng.TestRunner.run(TestRunner.java:623)
	at org.testng.SuiteRunner.runTest(SuiteRunner.java:357)
	at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:352)
	at org.testng.SuiteRunner.privateRun(SuiteRunner.java:310)
	at org.testng.SuiteRunner.run(SuiteRunner.java:259)
	at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
	at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:86)
	at org.testng.TestNG.runSuitesSequentially(TestNG.java:1185)
	at org.testng.TestNG.runSuitesLocally(TestNG.java:1110)
	at org.testng.TestNG.run(TestNG.java:1018)
	at org.testng.remote.RemoteTestNG.run(RemoteTestNG.java:111)
	at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:204)
	at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:175)
]]>
            </full-stacktrace>
          </exception> <!-- java.lang.NullPointerException -->
          <reporter-output>
          </reporter-output>
        </test-method> <!-- testEmptyTestCreation -->
        <test-method status="FAIL" signature="testNonEmptyTestCreation()[pri:0, instance:com.example.tests.CreateGroup@71f2a7d5]" name="testNonEmptyTestCreation" duration-ms="0" started-at="2015-12-20T22:20:58Z" finished-at="2015-12-20T22:20:58Z">
          <exception class="java.lang.NullPointerException">
            <full-stacktrace>
              <![CDATA[java.lang.NullPointerException
	at com.example.tests.CreateGroup.testNonEmptyTestCreation(CreateGroup.java:8)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
	at java.lang.reflect.Method.invoke(Method.java:497)
	at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:85)
	at org.testng.internal.Invoker.invokeMethod(Invoker.java:639)
	at org.testng.internal.Invoker.invokeTestMethod(Invoker.java:821)
	at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:1131)
	at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:124)
	at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:108)
	at org.testng.TestRunner.privateRun(TestRunner.java:773)
	at org.testng.TestRunner.run(TestRunner.java:623)
	at org.testng.SuiteRunner.runTest(SuiteRunner.java:357)
	at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:352)
	at org.testng.SuiteRunner.privateRun(SuiteRunner.java:310)
	at org.testng.SuiteRunner.run(SuiteRunner.java:259)
	at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
	at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:86)
	at org.testng.TestNG.runSuitesSequentially(TestNG.java:1185)
	at org.testng.TestNG.runSuitesLocally(TestNG.java:1110)
	at org.testng.TestNG.run(TestNG.java:1018)
	at org.testng.remote.RemoteTestNG.run(RemoteTestNG.java:111)
	at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:204)
	at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:175)
]]>
            </full-stacktrace>
          </exception> <!-- java.lang.NullPointerException -->
          <reporter-output>
          </reporter-output>
        </test-method> <!-- testNonEmptyTestCreation -->
      </class> <!-- com.example.tests.CreateGroup -->
    </test> <!-- Default test -->
  </suite> <!-- Default suite -->
</testng-results>
